code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -102,13 +102,13 @@ All foods are distributed among various categories. Use common sense.
 				to_chat(M, "<span class='notice'>You don't feel like eating any more junk food at the moment.</span>")
 				return 0
 			else if(fullness <= 50)
-				user.visible_message("<span class='notice'>[user] hungrily takes a [eatverb] from \the [src], gobbling it down!</span>", "<span class='notice'>You hungrily take a [eatverb] from \the [src], gobbling it down!</span>")
+				user.visible_message("<span class='notice'>[user] hungrily [eatverb] \the [src], gobbling it down!</span>", "<span class='notice'>You hungrily [eatverb] \the [src], gobbling it down!</span>")
 			else if(fullness > 50 && fullness < 150)
-				user.visible_message("<span class='notice'>[user] hungrily takes a [eatverb] from \the [src].</span>", "<span class='notice'>You hungrily take a [eatverb] from \the [src].</span>")
+				user.visible_message("<span class='notice'>[user] hungrily [eatverb] \the [src].</span>", "<span class='notice'>You hungrily [eatverb] \the [src].</span>")
 			else if(fullness > 150 && fullness < 500)
-				user.visible_message("<span class='notice'>[user] takes a [eatverb] from \the [src].</span>", "<span class='notice'>You take a [eatverb] from \the [src].</span>")
+				user.visible_message("<span class='notice'>[user] [eatverb] \the [src].</span>", "<span class='notice'>You [eatverb] \the [src].</span>")
```

Comment:
[user] slurp the coffee.

Reply:
Yes, you are right. The few that fit before no longer does, I will fix them.

Reply:
Or rather, the observer strings were wrong. Fixed.